docs/2024/puzzles/day13.md

```diff
@@ -6,10 +6,208 @@ import Solver from "../../../../../website/src/components/Solver.js"
 
 https://adventofcode.com/2024/day/13
 
+## Solution Summary
+
+This problem requires you to work with equations and numbers.
+
+Both parts of the problem asks you to calculate the smallest number of tokens needed to win as many prizes as possible, which means to calculate the optimal number of A and B button presses.
+
+- For part 1, the question assures that buttons are pressed no more than 100 times, so it can be simply brute forced for all possible combinations.
```

Comment:
```suggestion
- For part 1, the question assures you that buttons are pressed no more than 100 times, so it can simply be brute forced for all possible combinations.
```